packages/react/src/themes/teams/components/Button/buttonVariables.ts

```diff
@@ -132,5 +133,6 @@ export default (siteVars: any): ButtonVariables => {
     textSecondaryColorHover: siteVars.brand04,
 
     boxShadow: siteVars.shadowLevel1,
+    focusedBorderRadius: pxToRem(3),
```

Comment:
just to introduce the same order that is implied by classification we are, essentially, converging to, maybe we might want to mention state at the end (like `borderRadiusFocused`)?

![image](https://user-images.githubusercontent.com/9024564/52961728-aa88e300-33ac-11e9-98ad-2f34c03de247.png)

Reply:
Agreed, thanks!